WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueEditorContentView.swift

```diff
@@ -0,0 +1,102 @@
+import SwiftUI
+
+/// Prologue editor page contents
+struct UnifiedPrologueEditorContentView: View {
+
+    var body: some View {
+        GeometryReader { content in
+
+            VStack {
+                Spacer(minLength: content.size.height * 0.18)
+
+                RoundRectangleView {
+                    HStack {
+                        Text(Appearance.topElementTitle)
+                            .font(Font.system(size: content.size.height * 0.08,
+                                              weight: .semibold,
+                                              design: .serif))
+                        Spacer()
+                    }
+                    .padding(.all, content.size.height * 0.06)
+                }
+                .fixedSize(horizontal: false, vertical: true)
+
+                Spacer(minLength: content.size.height * 0.03)
+                    .fixedSize(horizontal: false, vertical: true)
+
+                RoundRectangleView(alignment: .top) {
+                    (Text(Appearance.middleElementTitle)
+                        + Text(Appearance.middleElementTerminator)
+                        .foregroundColor(.blue))
+                        .font(Font.system(size: content.size.height * 0.06,
+                                          weight: .regular,
+                                          design: .default))
+                        .fixedSize(horizontal: false, vertical: true)
+                        .lineLimit(.none)
+                        .padding(.all, content.size.height * 0.06)
+
+                    HStack {
+                        let alignImageLeftIconSize = content.size.height * 0.15
+                        CircledIcon(size: alignImageLeftIconSize,
+                                    xOffset: -alignImageLeftIconSize * 3 / 4,
```

Comment:
I wonder if it would be clearer to write these as * 0.75, like most of the other ratios in here?